Formula/libgit2.rb

```diff
@@ -48,7 +50,7 @@ def install
 
       int main(int argc, char *argv[]) {
         int options = git_libgit2_features();
-        return 0;
+        return (options & GIT_FEATURE_SSH) > 0;
```

Comment:
Are we expecting `(options & GIT_FEATURE_SSH) > 0` to be false(-y)?

Reply:
Oops you are right, I changed the syntax in the last minute and then broke the test. I'll change it to assert.